airbyte-config/init/src/main/resources/seed/source_specs.yaml

```
@@ -6759,7 +6759,7 @@
     supportsNormalization: false
     supportsDBT: false
     supported_destination_sync_modes: []
-- dockerImage: "airbyte/source-sendgrid:0.2.6"
+- dockerImage: "airbyte/source-sendgrid:0.3.0"
```

```suggestion
- dockerImage: "airbyte/source-sendgrid:0.2.7"
```
adding a new stream is not breaking or adding any new features for the connector.